doc/source/serve/llm/serving-llms.rst

```diff
@@ -513,22 +513,17 @@ For example, if you have a model stored in S3 that looks like the below structur
 
 .. code-block:: bash
 
-    $ aws s3 ls s3://rob-general-test-bucket/meta-llama/
-                               PRE original/
-    2025-03-21 16:26:32       1519 .gitattributes
-    2025-03-21 16:28:02       7627 LICENSE
-    2025-03-21 16:29:35      44044 README.md
-    2025-03-21 16:28:46       4691 USE_POLICY.md
-    2025-03-21 16:27:32        855 config.json
-    2025-03-21 16:27:52        184 generation_config.json
-    2025-03-21 16:28:15 4976698672 model-00001-of-00004.safetensors
-    2025-03-21 16:30:48 4999802720 model-00002-of-00004.safetensors
-    2025-03-21 16:31:24 4915916176 model-00003-of-00004.safetensors
-    2025-03-21 16:30:32 1168138808 model-00004-of-00004.safetensors
-    2025-03-21 16:30:01      23950 model.safetensors.index.json
-    2025-03-21 16:29:24        296 special_tokens_map.json
-    2025-03-21 16:29:00    9085657 tokenizer.json
-    2025-03-21 16:29:13      55351 tokenizer_config.json
+    $ aws s3 ls s3://air-example-data/meta-Llama-3.2-1B-Instruct/
```

Comment:
sorry, I wasn't clear earlier. Can you put it here: `s3://air-example-data/rayllm-ossci/` ? On legacy account? For anonymizing role you can use `s3://anonymous@air-example-data/rayllm-ossci/`